src/utils/constants.ts

```diff
@@ -14,3 +14,5 @@ export const OPENSEA_API_KEY = process.env.REACT_APP_OPENSEA_API_KEY || ''
 export const COLLECTIBLES_SOURCE = process.env.REACT_APP_COLLECTIBLES_SOURCE || 'OpenSea'
 export const TIMEOUT = process.env.NODE_ENV === 'test' ? 1500 : 5000
 export const ETHERSCAN_API_KEY = process.env.REACT_APP_ETHERSCAN_API_KEY
+export const SPENDING_LIMIT_MODULE_ADDRESS =
+  process.env.REACT_APP_SPENDING_LIMIT_MODULE_ADDRESS || '0x9e9Bf12b5a66c0f0A7435835e0365477E121B110'
```

Comment:
This address should be named. I understand this is something will be removed later

Reply:
exactly, this is there to make the feature work while on dev, as the variable is not set in the CI builds

Reply:
I created a task in [Backlog] - Stability saying that this should be reviewed before deployment